website/docs/r/budgets_budget.html.markdown

```diff
@@ -58,6 +58,46 @@ resource "aws_budgets_budget" "s3" {
 }
 ```
 
+Create a Savings Plan Utilization Budget
+```hcl
+resource "aws_budgets_budget" "savings_plan_utilization" {
+  # ...
+  budget_type  = "SAVINGS_PLANS_UTILIZATION"
+  limit_amount = "100"
```

Comment:
To prevent perpetual differences when using the `PERCENTAGE` unit, it seems this needs to be `"100.0"`

```suggestion
  limit_amount = "100.0"
```